kyaml/yaml/rnode.go

```diff
@@ -474,13 +474,32 @@ func (rn *RNode) ElementValues(key string) ([]string, error) {
 	return elements, nil
 }
 
+func (rn *RNode) ElementValuesList(keys []string) ([][]string, error) {
+	if err := ErrorIfInvalid(rn, yaml.SequenceNode); err != nil {
+		return nil, errors.Wrap(err)
+	}
+	elements := make([][]string, len(rn.Content()))
+
+	for i := 0; i < len(rn.Content()); i++ {
+		for _, key := range keys {
+			field := NewRNode(rn.Content()[i]).Field(key)
+			if !field.IsNilOrEmpty() {
+				elements[i] = append(elements[i], field.Value.YNode().Value)
+			} else {
+				elements[i] = append(elements[i], "")
+			}
+		}
+	}
+	return elements, nil
+}
+
 // Element returns the element in the list which contains the field matching the value.
 // Returns nil for non-SequenceNodes or if no Element matches.
-func (rn *RNode) Element(key, value string) *RNode {
+func (rn *RNode) Element(keys []string, values []string) *RNode {
```

Comment:
same question: will this passing arg change break existing caller functions?

Reply:
`RNode` is one of the most fundamental struct in kyaml. We need to have tons of changes to apply this modification. And I believe this change will break kpt somehow.